src/pages/Vote/VotingWallet/index.tsx

```diff
@@ -1,20 +1,82 @@
 /** @jsxImportSource @emotion/react */
 import React from 'react';
-import { Typography } from '@mui/material';
+import BigNumber from 'bignumber.js';
+import { Paper, Typography } from '@mui/material';
+import { Link } from 'react-router-dom';
+import { Delimiter, Icon, LinkButton } from 'components';
+import PATHS from 'constants/path';
 import { useTranslation } from 'translation';
+import { convertWeiToCoins, format } from 'utilities/common';
 import { useStyles } from './styles';
 
-export const VoteUi: React.FC = () => {
-  const { t } = useTranslation();
-  const styles = useStyles();
+interface IVoteUiProps {
+  votingWeight: BigNumber;
+  xvsLockedWei: BigNumber;
+}
 
+export const VoteUi: React.FC<IVoteUiProps> = ({ votingWeight, xvsLockedWei }) => {
+  const { t, Trans } = useTranslation();
+  const styles = useStyles();
+  const readableXvsLocked = convertWeiToCoins({
+    valueWei: xvsLockedWei,
+    tokenId: 'xvs',
+    returnInReadableFormat: true,
+    addSymbol: false,
```

Comment:
We'll also need to minimize decimals:
```suggestion
    addSymbol: false,
    minimizeDecimals: true,
```